### ISSUE TITLE
Accessibility Issues with Grid Excel Style Filter dialog

### ISSUE THREAD

**Issue by ericmahlberg** [IgniteUI/igniteui-angular#9287]
## Description  
Accessibility issues with the igx-Grid Excel Style filtering dialog and sub dialogs.

igniteui-angular version: as early as 10.1.9 through to the latest 11.x
browser: Chrome

## Steps to reproduce  

1. Navigate to https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering
2. Open one of the column filter dialogues so that the excel like options are visible, including something like Date filters
3. Run an accessibility tool against the page (https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni)

## Result  
What is the actual result after following the steps to reproduce?
Accessibility issues pertaining to buttons having discernible text, sub menu items not having particular aria tags, keyboard accessible

Title: WCAG 4.1.2: Ensures buttons have discernible text (.igx-button--icon[igxbutton="icon"][ng-reflect-type="icon"]:nth-child(2))
Tags: Accessibility, WCAG 4.1.2, button-name
Issue: Ensures buttons have discernible text (button-name - https://accessibilityinsights.io/info-examples/web/button-name)
Element path: .igx-excel-filter__menu-header-actions > .igx-button--icon[igxbutton="icon"][ng-reflect-type="icon"]:nth-child(2)
Snippet: button igxbutton="icon" class="igx-button--icon ng-star-inserted" ng-reflect-type="icon" role="button"
How to fix:
Fix any of the following:
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element's default semantics were not overridden with role="none" or role="presentation"
Element has no title attribute

Title: WCAG 4.1.2: Ensures every ARIA input field has an accessible name (#igx-drop-down-3-list)
Tags: Accessibility, WCAG 4.1.2, aria-input-field-name
Issue: Ensures every ARIA input field has an accessible name (aria-input-field-name - https://accessibilityinsights.io/info-examples/web/aria-input-field-name)
Element path: #igx-drop-down-3-list
Snippet: div role="listbox" class="igx-drop-down__list-scroll" id="igx-drop-down-3-list" style="max-height: 397px;"
How to fix: 
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

Title: WCAG 2.1.1: Elements that have scrollable content should be accessible by keyboard (#igx-drop-down-3-list)
Tags: Accessibility, WCAG 2.1.1, scrollable-region-focusable
Issue: Elements that have scrollable content should be accessible by keyboard (scrollable-region-focusable - https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable)
Element path: #igx-drop-down-3-list
Snippet: div role="listbox" class="igx-drop-down__list-scroll" id="igx-drop-down-3-list" style="max-height: 397px;"
How to fix: 
Fix any of the following:
  Element should have focusable content
  Element should be focusable

## Expected result  
No accessibility issues

## Attachments  
![image](https://user-images.githubusercontent.com/7574313/113906393-0c715600-9789-11eb-82f4-5321c7ffbb8b.png)
![image](https://user-images.githubusercontent.com/7574313/113906413-12673700-9789-11eb-9cb5-5d54c241746d.png)
![image](https://user-images.githubusercontent.com/7574313/113949500-47dd4600-97c4-11eb-9728-3af64f749819.png)
![image](https://user-images.githubusercontent.com/7574313/113950223-ace56b80-97c5-11eb-9d15-e88017789ffa.png)
![image](https://user-images.githubusercontent.com/7574313/113949523-57f52580-97c4-11eb-91f3-3fe455ebecce.png)
![image](https://user-images.githubusercontent.com/7574313/113949610-86730080-97c4-11eb-8473-3dcd78916dce.png)
